### DISCUSSION TITLE
Come puntare al nuovo spazio di hosting del sito

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/opendatasicilia.it#18] [Generale]
Per un po' di tempo il l'URL di opendatasicilia, puntare al vecchio sito, in modo da fare delle verifiche e un po' di pulizia sul nuovo spazio, dove è stato già importato tutto.

Per navigare sul nuovo spazio, usando www.opendatasicilia.it, seguire queste istruzioni:

- per win https://www.vhosting-it.com/clients/knowledgebase/84/Come-visualizzo-il-sito-che-e-in-fase-di-trasferimento.html
- per mac https://www.vhosting-it.com/documentazione/come-visualizzo-il-sito-che-e-in-trasferimento-mac/

In Windows ad esempio basta aggiungere nel file `hosts`

```
86.105.14.14 opendatasicilia.it www.opendatasicilia.it
```

**Comment by gbvitrano** [comment]
in questa fase è possibile entrare come admin in WordPress. e installare il template provvisorio? 
io posso farlo?
l'indirizzo per il login?
![2021-11-27_17h43_37](https://user-images.githubusercontent.com/20488693/143689716-fc0f0882-073c-434d-baf0-ee8c0f938af9.png)

**Reply by aborruso** [reply]
Caro @gbvitrano l'URL è il classico, con `wp-admin` alla fine.

E sì, installa quello che vuoi. Se non funziona, apri issue

**Comment by gbvitrano** [comment]
Ok grazie 😊


Giovan Battista Vitrano - mobile - @gbvitrano

Il sab 27 nov 2021, 17:52 Andrea Borruso ***@***.***> ha
scritto:

**Comment by dennisangemi** [comment]
![image](https://user-images.githubusercontent.com/77018886/143719392-805f0f72-b366-4cf0-96f7-54cf9ba3deff.png)
Ho provato a modificare il file hosts ma non riesco a salvarlo (nonostante io sia l'amministratore del mio pc) :(

**Reply by dennisangemi** [reply]
Risolto: ho modificato le impostazioni di protezione dando accesso a tutti gli utenti del pc (per qualche strano motivo mi considerava un semplice utente e non amministratore)

**Reply by aborruso** [reply]
Devi lanciare il tuo editor di testo da administrator.

Se usi notepad++ (mai più senza), e apri quel file normalmente, e salvi, ti verrà chiesto "vuoi riaprire il programma da amministratore?".

**Reply by dennisangemi** [reply]
Io non ho installato notepad++, ho usato solo "blocco note"

**Reply by aborruso** [reply]
Ti facevo solo un esempio e condividevo un tips and tricks.

Sappi però che tutte le volte che un utente usa il blocco note di Windows per editare un file, muore un 🦄

**Reply by dennisangemi** [reply]
Sappi che è già partito il download di notepad++ allora haha
